### ISSUE TITLE
[1.2.0 Beta 1]Password change during install

### ISSUE THREAD

**Issue by jpobeda** [Cacti/cacti#2086]
Hi there,

I git pulled to make sure I was running the latest code

When I got to this point

![firefox_2018-10-18_17-48-09](https://user-images.githubusercontent.com/13456559/47132353-f133d700-d2ff-11e8-8983-cffa943a851e.png)

I know what is going to happen, I introduce default admin password and then a dummy password like 'dummydummy0'

At this point the installer never told what was the password policy in place. But it looks "green" and good to go.

![firefox_2018-10-18_17-48-33](https://user-images.githubusercontent.com/13456559/47132303-a6b25a80-d2ff-11e8-83ca-b4312a5cdc77.png)

So I get this after I hit save (it complains about Mix case **only**)

![firefox_2018-10-18_17-48-50](https://user-images.githubusercontent.com/13456559/47132308-add96880-d2ff-11e8-986e-a891381e3e2e.png)

Damn..OK so I tried 'DummyDummy0' and again all green and good to go.

BUT..I got

![firefox_2018-10-18_17-49-33](https://user-images.githubusercontent.com/13456559/47132324-c9dd0a00-d2ff-11e8-85fc-96f1b2cb019e.png)

So long story short, I think the password policy should be printed the very first time you ask the user to change the default password and also if you're going to check that the password meets the policy or not, why doing it in small bits? It should have told me the first time that I also needed the special character.

**Comment by cigamit** [comment]
Correct, and it should work as advertised ;)

**Comment by cigamit** [comment]
Resolved.  Thanks for catching.
